app/_kong_plugins/request-termination/index.md

```diff
@@ -31,6 +30,37 @@ categories:
 
 search_aliases:
   - request-termination
+
+related_resources:
+  - text: Allow clients to choose their authentication methods and prevent unauthorized access
+    url: /how-to/allow-multiple-authentication/
 ---
 
-## Overview
+This plugin terminates incoming requests with a specified status code and message.
+This can be used to temporarily stop traffic on a Gateway Service or a Route, or even block a Consumer. 
+
+This plugin can also be used for debugging, as described in the [`config.echo` parameter](/plugins/request-termination/reference/#schema--config-echo).
+
+Once this plugin is enabled, every request within the configured scope will be immediately terminated with the configured status code and message.
+
+## Request termination priority
+
+This plugin has a priority of `2`, which is the lowest priority of all plugins except Post-Function. This means it will execute after all other plugins have been applied. 
```

Comment:
```suggestion
This plugin has a priority of `2`, which is the lowest [priority of all plugins](/gateway/entities/plugin/#plugin-priority) except Post-Function. This means it will execute after all other plugins have been applied. 
```